integreat_cms/core/settings.py

```diff
@@ -108,6 +108,13 @@
     "INTEGREAT_CMS_WIKI_URL", "https://wiki.integreat-app.de"
 )
 
+#: Platform Name, used for app users and social media headers
+
+#: Social media preview image
+SOCIAL_PREVIEW_IMAGE = os.environ.get(
+    "INTEGREAT_SOCIAL_PREVIEW_IMAGE", f"{WEBAPP_URL}/social-media-preview.png"
+)
```

Comment:
Jep. That would make sense. Even better if this can be configured per page at some point. In case of pois & events we actually could already use the thumbnail image. What do you think?

Reply:
Cool idea, yes. Do we need to generate the thumbnails in a specific aspect ratio for the social media previews? At the moment, they're just 1:1...

Reply:
https://stackoverflow.com/a/15560644 I guess this says enough about the size. But you can also specify the size of the page https://ogp.me/#structured

Reply:
I also just saw that Malte **does** have a different social media preview image. Therefore, I might have to move some stuff around again.